docs/examples/model_repository/resnet_torchscript/Readme.md

```diff
@@ -0,0 +1,154 @@
+# Loading Pytorch Models To Nvidia Triton Inference Server
+
+Pytorch models are supported by the Nvidia [Triton Inference Server](https://github.com/triton-inference-server/). To Load the Pytorch model to triton inference server one must convert the model either ONNX format or in the TorchScript format.
+Here we will see how to convert the Pytorch Model to the Torchscript format and load it to Triton.
+
+## Imports 
```

Comment:
Create a single section **Converting a PyTorch model defined in Python to TorchScript**.
If you want to describe what TorchScript is it should be done here before the example below.   

Instead of breaking up the code like this I would create two segments: 
1. Create and load the model.
2. Create test data, trace the model and save as a model.pt file. 

Avoid slowing the logs or results here as.